### PR TITLE
Connor dev

### PR DESCRIPTION
**Pull request by connorbrett**
image
Added a background image, stylized the profile picture, and added border

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
  */
+document.getElementsByTagName("h1")[0].style.fontSize = "6vw";
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+  console.log("greeting: " + greeting);
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js:25]: nit: Left over from debugging?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
+document.getElementsByTagName("h1")[0].style.fontSize = "6vw";
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js:18]: This seems already covered by CSS. Is there a reason you're setting it via JavaScript as well?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
+    <title>Connor's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <div class="card">
+      <h1 fontSize = "6vw">Connor's Portfolio</h1>
```
Comment (EatingW) [inline portfolio/src/main/webapp/index.html:11]: One more nit: According to https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, let's keep styling in css files and have the HTML only be for structure purposes.
Same for the line below for your avatar.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (connorbrett) [reply portfolio/src/main/webapp/script.js:25]: yep, was part of the intro exercise they had us do

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (connorbrett) [reply portfolio/src/main/webapp/script.js:18]: saw it in an example online but realized i could just do it on the css itself

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (connorbrett) [reply portfolio/src/main/webapp/index.html:11]: is there a problem with line 10 or just line 11?

Comment (EatingW) [reply portfolio/src/main/webapp/index.html:11]: Huh, I don't know why github showed both line 10 and 11. The comment is actually about lines 11 and 12 lol

Comment (connorbrett) [reply portfolio/src/main/webapp/index.html:11]: I understand the issue with this line: " <h1 fontSize = "6vw">Connor's Portfolio</h1>". Is there an issue with this line: "<div class="card">"?

**Pull request merged**